app/src/main/java/com/example/mytrainingpal/components/Break.kt

```diff
@@ -1,4 +1,4 @@
-package com.example.mytrainingpal.screens
```

Comment:
why components? screen should stay screen or become `activities`

Reply:
The renaming I requested was `components` -> `composable`

Reply:
Because the Break is no screen but a component that gets used by the ToggleScreen

Reply:
Oh, I see, you moved the file into a different directory? 🤔

Reply:
Exactly :)
(Raissa already moved it but did not adjust the package name)